pkg/app/application.go

```diff
@@ -104,6 +105,10 @@ func (a *Application) loadCustomizedResMap() (resmap.ResMap, error) {
 	if err != nil {
 		errs.Append(errors.Wrap(err, "rawResources"))
```

Comment:
please change _rawResources_ to _loadResMapFromBasesAndResources_

(my mistake)